### DISCUSSION TITLE
ST7789 240x280, RP2040, 8bit parallel testing

### DISCUSSION THREAD

**Discussion by joey120373** [Bodmer/TFT_eSPI#2567] [General]
Buy display offers the 1.69 inch 240x280 rounded corner display with a 18 pin connector, set up for 8 bit parallel communication.
I started with an RP2040 to test, I may try an ESP32 mico module next...
RP2040 clocked at 150MHz works fine, 200MHz causes issues.

made a short video of the results :
https://youtube.com/shorts/uVtKiDHy_TQ?feature=share

**Comment by Bodmer** [comment]
Glad to hear you have it working. The display interface speed will need to be reduced at high CPU clock speeds. See here:
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setup.h#L327
